pr_agent/algo/pr_processing.py

```diff
@@ -353,9 +353,9 @@ async def retry_with_fallback_models(f: Callable, model_type: ModelType = ModelT
                 raise Exception(f"Failed to generate prediction with any model of {all_models}")
 
 
-def _get_all_models(model_type: ModelType = ModelType.REGULAR) -> List[str]:
-    if model_type == ModelType.TURBO:
-        model = get_settings().config.model_turbo
+def _get_all_models(model_type: ModelType = ModelType.WEAK) -> List[str]:
+    if model_type == ModelType.WEAK:
+        model = get_settings().config.model_weak
```

Comment:
the default model should be the regular model.
Also, we talked about utilizing this only if two conditions are met:
- a weak model is defined
- the tool asks for a weak model.

So something like:

```python
if get_settings().config.get('model_weak') and model_type == ModelType.WEAK:
  model = get_settings().config.model_weak
else:
  model = get_settings().config.model
```

Reply:
note that PR-Agent gave a similar suggestion :)

![image](https://github.com/user-attachments/assets/ec3c2686-3752-44a1-b743-1cd56954b346)

Reply:
Good catch, I will adjust the code accordingly.